nuxeo-core/nuxeo-core-storage/src/main/java/org/nuxeo/ecm/core/storage/BaseDocument.java

```diff
@@ -273,7 +278,7 @@ protected boolean checkReadOnlyIgnoredWrite(Property property, T state) throws P
         if (!isVersion()) {
             throw new PropertyException("Cannot write readonly property: " + name);
         }
-        if (!name.startsWith(DC_PREFIX)) {
+        if (!name.startsWith(DC_PREFIX) && !NOTIFICATION_SCHEMA.equals(property.getSchema().getName())) {
```

Comment:
Why not do, similarly to what we do for dublincore, just `name.startsWith(NOTIF_PREFIX)`

Reply:
Because for sub properties we don't have prefix in the name. I try to do it, but it added too many properties in `VERSION_WRITABLE_PROPS`.